### ISSUE TITLE
Library tries to fetch a favicon.ico several times per dispatch

### ISSUE THREAD

**Issue by langerhans** [Tealium/tealium-android#9]
The library seems to request a favicon.ico from the tag server several times per dispatch. Disregarding the fact how abstruse the whole hidden webview thing is in first place, this is still bad as it produces unneeded traffic for the users (and on your servers).

See following log:
```
D/Tealium-5.0.4: Sent new dispatch [...]
E/Tealium-5.0.4: Received http error with https://tags.tiqcdn.com/favicon.ico: 403; Forbidden
E/Tealium-5.0.4: Received http error with https://tags.tiqcdn.com/favicon.ico: 403; Forbidden
E/Tealium-5.0.4: Received http error with https://tags.tiqcdn.com/favicon.ico: 403; Forbidden
```
You can just place the following line in your HTML downloaded by the webview to prevent this: 
```html
<link rel="icon" type="image/png" href="data:image/png;base64,iVBORw0KGgo=">
```
Source: https://stackoverflow.com/questions/1321878/how-to-prevent-favicon-ico-requests#comment50777781_13416784

**Comment by craigrouse** [comment]
Hi @langerhans 

Sorry for the late reply on this. We're looking into solving this issue in a future release.

Regarding the webview generally, it's worth noting that we do also support a server-side approach (we call it "Cloud Delivery"), and if you go down this route, then it's possible to disable the webview entirely. This can be done without changing anything within your app; the API remains the same whether you use Cloud Delivery, Webview, or both. Please get in touch with your account manager if you'd like to learn more.

We'll keep this issue open and close it off when we have a resolution.

**Comment by langerhans** [comment]
Thank you for pointing out this alternative. I'll get in touch with our analytics team about it 👍 

Have a nice weekend!

**Comment by craigrouse** [comment]
@langerhans Sorry it's taken us a while to get this release out, but we have fixed this in the latest release (5.3.1). I'm closing this issue now.
